cypress/features/regression/test/actionPopoverNoIFrame.feature

```diff
@@ -42,20 +42,21 @@ Feature: Action Popover component in noIFrame
 
   @positive
   Scenario: verify that the first element Business is focused using Home key
-    When I click the menu button element in noiFrame
-      And I press Home on focused element
+    Given I click the menu button element in noiFrame
+      And I press keyboard downarrow key times 2
+    When I press Home on focused element
     Then focused element inner content is set to "Business"
       And Action Popover element has golden border on focus
 
   @positive
-  Scenario: verify that the first element sub menu 1 is focused using Home key for submenu
-    When I click the menu button element in noiFrame
+  Scenario: verify that the first sub menu 1 element is focused using Home key for submenu
```

Comment:
```suggestion
  Scenario: verify that the first sub menu element is focused using Home key for submenu
```

Reply:
I've clarified that the `sub menu 1` is focused